lib/bugsnag/configuration.rb

```diff
@@ -35,12 +34,26 @@ class Configuration
     attr_accessor :ignore_classes
     attr_accessor :auto_capture_sessions
     attr_accessor :track_sessions
-    attr_accessor :session_endpoint
+
+    ##
+    # @return [String] URL error notifications will be delivered to
+    attr_reader :notify_endpoint
+    alias :endpoint :notify_endpoint
+
+    ##
+    # @return [String] URL session notifications will be delivered to
+    attr_reader :session_endpoint
+
+    ##
+    # @return [Boolean] whether any sessions types will be delivered
+    attr_reader :send_sessions
```

Comment:
Having a boolean setting/option/attribute with the same name as a public method in `SessionTracker` (`send_sessions`) might be slightly confusing. It might be preferable to rename this option to something like `sessions_enabled` or `session_tracking_enabled` instead.